leapp/messaging/answerstore.py

```diff
@@ -24,7 +24,7 @@ def answer(self, scope, key, value):
         # So don't even bother updating this to some more 'pythonic' coding style
         self._storage[scope] = dialog_scope
 
-    def update(self, answer_file):
+    def update(self, answer_file, allow_missing=False):
```

Comment:
Somehow keeping allow_missing here makes more sense to me than renaming that to add (tribute to configparser's allow_no_value?).

Reply:
Yeah I saw that and it's fine, it's just shorter to write --add than allow-missing on the command line if typed manually